src/java/biomes/Biomes.java

```diff
@@ -0,0 +1,69 @@
+package biomes;
+
+import com.google.gson.Gson;
+import lombok.Data;
+import map.Material;
+import map.TerrainMaterials;
+
+import java.io.IOException;
+import java.nio.file.DirectoryStream;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.ArrayList;
+import java.util.List;
+
+@Data
+public strictfp class Biomes {
+
+	public static List<TerrainMaterials> terrainMaterials = new ArrayList<>();
+
+	public static void load(){
+
+		Path biomePath = null;
+		try {
+			biomePath = Paths.get(Biome.class.getClassLoader().getResource("custom_biomes").toURI());
+		}
+		catch (Exception e){
+			e.printStackTrace();
```

Comment:
I'd say print an error message and quit the generator. If biomePath is null we will get an exception further down otherwise.

Reply:
(additionally to the stack trace)

Reply:
Please print an error message additionally to the stack trace. (no logger here, just use `System.err.println();`)